runtime/manifest.js

```diff
@@ -219,6 +230,40 @@ class Manifest {
           connection.connectToView(targetView);
         }
       }
+      for (let slotConnectionItem of item.slotConnections) {
+        let slotConn = particle.addSlotConnection(slotConnectionItem.param, slotConnectionItem.dir);
+        if (!slotConnectionItem.name) {
+          continue;
+        }
+        assert(items.byName.has(slotConnectionItem.name));
+        let {item, slot} = items.byName.get(slotConnectionItem.name);
+        slotConn.connectToSlot(slot);
+        if (item.ref.view || (item.ref.tags.length > 0)) {
+          let foundConnection = false;
+          for (let particleViewConn of Object.values(particle.connections)) {
+            if (!particleViewConn.view) continue;
+            if ((particleViewConn.view.localName == item.ref.view) ||
+                (item.ref.tags.every(t => particleViewConn.view.tags.indexOf(t) >= 0))) {
+              slotConn.connectToView(particleViewConn.name);
+              foundConnection = true;
+              break;
+            }
+          }
+          // add new unnamed connections
+          if (!foundConnection) {
+            let viewConn = particle.addUnnamedConnection();
```

Comment:
i see now that i am indeed missing a view in the recipe Slot. 
then a Particle that provides this slot, needs its SlotConnection to have the ViewConnection that is resolved to that view in the slot.
Does that sound reasonable?